private/manageFishData.py

```diff
@@ -349,6 +349,17 @@ def is_big_fish(fish_id):
     return False
 
 
+def get_min_collectability(fish_id):
+    try:
+        shop_item = next(filter(lambda row: row.get_raw('Item') == fish_id,
+                         XIV.game_data.get_sheet('CollectablesShopItem')))
+    except StopIteration:
+        shop_item = None
```

Comment:
I think `next` can take an optional default argument to avoid catching exceptions.
Also, the second line technically should be indented to align with filter.